test/source/tests/tooling/browser-recipe.ts

```diff
@@ -100,6 +100,15 @@ export class BrowserRecipe {
     return (result as { result: string }).result;
   };
 
+  public static getSavedPassPhrase = async (controllable: Controllable, acctEmail: string, longid: string): Promise<string> => {
```

Comment:
Good.
However, `getSavedPassPhrase`  is abmiguous, as a pass phrase can be stored in the local storage too.
So this method should be named in a way that indicates the in-memory store is queried,
something like `getPassphraseFromInMemoryStore`